extensions/wikia/EmbeddableDiscussions/scripts/DiscussionsSettingsModal.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * Modal containing settings for the Discussions Module
+ */
+define('DiscussionsSettingsModal',
+	['jquery', 'wikia.loader', 'mw', 'wikia.mustache', 'wikia.tracker', 'wikia.nirvana'],
+	function ($, loader, mw, mustache, tracker, nirvana) {
+		'use strict';
+		var modalConfig = {
+				vars: {
+					id: 'DiscussionsSettingsModal',
+					classes: ['embeddable-discussions-settings-modal'],
+					size: 'small'
+				}
+			},
+			track = tracker.buildTrackingFunction({
+				action: tracker.ACTIONS.CLICK,
+				category: 'community-page-discussions-settings-modal',
+				trackingMethod: 'analytics'
+			}),
+			config;
+
+		function saveSettings() {
+			var $selectedCategories = $('.embeddable-discussions-filter-by-list input:checked');
```

Comment:
I'd create variables with all jquery objects (elements), especially for those which are used more than one time.